assembl/static2/js/app/index.jsx

```diff
@@ -10,17 +10,17 @@ import createAppStore from './store';
 import client from './client';
 import Routes from './routes';
 
-
 /*
 `piwik-react-router` expects that we provide it with a browser history.
 We provide it with `react-router`'s `browserHistory`.
 This is not the official way of doing it.
 The official way is using the `history` library, called with `import history from './utils/history';`.
-For details, see https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
+For details, see
+https://github.com/ReactTraining/react-router/blob/master/FAQ.md#how-do-i-access-the-history-object-outside-of-components
 */
 const history = browserHistory;
 
-const store = createAppStore(client);
+const store = createAppStore();
```

Comment:
It's to not include apollo cache in the redux store. Apollo will create its own redux store to manage its cache. This is to avoid unncessary call to mapStateToProps of redux connect containers when the apollo store is updated when receiving a query response.
Ah, I just said what I said in my commit actually. Is is clearer?